test/examples/ucp_client_server.c

```diff
@@ -649,14 +663,29 @@ static int run_server(ucp_context_h ucp_context, ucp_worker_h ucp_worker,
             goto err_listener;
         }
 
-        client_server_communication(ucp_data_worker, server_ep, send_recv_type, 1);
+        /* The server waits for all the iterations to complete before moving on
+         * to the next client */
+        while (server_iters_cnt < (iters + 1)) {
```

Comment:
start from 0
compare to iters without +1

Reply:
i started from 1 for the prints - to start from iteration 1 and not 0. i'll make the change in the printing then

Reply:
yes